src/conf_macros.m4

```diff
@@ -520,6 +520,7 @@ AC_DEFUN([WITH_LIBNL],
             AC_MSG_ERROR([Libnl required, but not available])
         fi
     fi
+    AM_CONDITIONAL([HAVE_LIBNL], test x"$HAVE_LIBNL" = x1)
```

Comment:
I'm not fluent with 'm4' at all, but looking at other examples like:
```
AM_CONDITIONAL([HAVE_SYSV], [test x"$initscript" = xsysv])
```
  --  shouldn't `test x"$HAVE_LIBNL" = x1` be in `[]`?

Reply:
I'm as fluent as you in `m4` and all the declarations do have the brackets. However, they don' t seem to be required as the condition works flawlessly without them.

I have added them anyway, for the sake of coherency.